tensorflow/core/kernels/data/experimental/random_dataset_op_test.cc

```diff
@@ -0,0 +1,269 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "tensorflow/core/kernels/data/experimental/random_dataset_op.h"
+
+#include "tensorflow/core/kernels/data/dataset_test_base.h"
+#include "tensorflow/core/lib/random/philox_random.h"
+#include "tensorflow/core/lib/random/random.h"
+#include "tensorflow/core/lib/random/random_distributions.h"
```

Comment:
error: module //third_party/tensorflow/core/kernels/data/experimental:random_dataset_op_test does not depend on a module exporting 'third_party/tensorflow/core/lib/random/random_distributions.h'

Reply:
The dependency on the `//tensorflow/core/lib/random:philox` that I added in in commit 1d547b2 includes `random_distributions.h`.